plugins/agent_types/AcquiaLiftTarget.inc

```diff
@@ -128,7 +142,12 @@ class AcquiaLiftTarget extends PersonalizeAgentBase implements PersonalizeExplic
         '!what_url' => url('admin/structure/personalize/manage/' . $this->getMachineName() . '/variations'),
       ));
     }
-    $errors += $this->getMultipleOptionSetErrors($option_sets);
```

Comment:
The function it calls utilizes the lock_step validation function in the module (outside of the class).  I had thought I moved it all back too, though.